src/Signup.jsx

```diff
@@ -0,0 +1,184 @@
+              </h1>
+
+              <div className="text-[16px] flex items-center gap-[8px] font-[400] justify-center">
+                <span className="signup_span" />
+                <p>
+                  Already have an account?{" "}
+                  <Link to="/" className="text-[#37BBCA]">
+                    {" "}
+                    {/* PATH TO LOGIN PAGE */}
+                    Login
+                  </Link>
+                </p>
+                <span className="signup_span" />
+              </div>
+            </div>
+
+            <form action="" className="py-4 pt-8 flex flex-col gap-y-6">
+              <input
+                type="email"
+                id="email"
+                name="email"
+                placeholder="Email"
+                className="auth_input"
+              />
+
+              <input
+                type="text"
+                id="firstName"
+                name="firstName"
+                placeholder="First name"
+                className="auth_input"
+              />
+
+              <input
+                type="text"
+                id="LastName"
+                name="LastName"
+                placeholder="Last name"
+                className="auth_input"
+              />
+
+              <select
+                name="country"
+                id="country"
+                className="auth_input text-[#848484] capitalize"
+              >
+                {Countries.map((country, index) => (
+                  <option value={country} key={index}>
+                    {country}
+                  </option>
+                ))}
+              </select>
+
+              <select
+                name="city"
+                id="city"
+                className="auth_input text-[#848484] capitalize"
+              >
+                {Cities.map((city, index) => (
+                  <option value={city} key={index}>
+                    {city}
+                  </option>
+                ))}
+              </select>
+
+              <input
+                type="number"
+                id="phoneNumber"
+                name="phoneNumber"
+                placeholder="Phone number"
+                className="auth_input"
+              />
+
+              <div className="relative">
+                <input
+                  type={!show ? "password" : "text"}
+                  id="password"
+                  name="password"
+                  placeholder="Password"
+                  className="auth_input"
+                />
+
+                <div className="absolute top-[10px] text-[#848484] cursor-pointer right-3 text-3xl">
+                  <BiShow
+                    className={show ? "hidden" : "block"}
+                    onClick={() => setShow(true)}
+                  />
+                  <BiSolidHide
+                    className={!show ? "hidden" : "block"}
+                    onClick={() => setShow(false)}
+                  />
```

Comment:
Instead of this, why not just do ...

```showPassword ? <BiShow onClick={() => setShowPassword(!showPassword)} /> : <BiSolidHide onClick={() => setShowPassword(!showPassword)} />```